xml/System.Numerics/BitOperations.xml

```diff
@@ -14,7 +14,7 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
+    <summary>Utility methods for intrinsic bit-twiddling operations. The methods use hardware intrinsics when available on the underlying platform, otherwise they use optimized software fallbacks.</summary>
     <remarks>To be added.</remarks>
```

Comment:
Is this staying `To be added.` going to be a problem?

Reply:
Remarks are optional. We only use them when it's necessary to explain something additional that doesn't fit in a summary, returns, param or typeparam field.
"To be added" is the default string for what dotnet-api-docs considers empty or undocumented. But in the case of remarks, since they are optional, it won't be a problem.

Reply:
"To be added." is automatically ignored at documentation build time.